docs/sources/administration/data-source-management/index.md

```diff
@@ -81,51 +82,47 @@ You can assign data source permissions to users, teams, and roles which will all
 
 <div class="clearfix"></div>
 
-## Query caching
+## Query and resource caching
 
-When query caching is enabled, Grafana temporarily stores the results of data source queries. When you or another user submit the exact same query again, the results will come back from the cache instead of from the data source (like Splunk or ServiceNow) itself.
+When query and resource caching is enabled, Grafana temporarily stores the results of data source queries and resource requests. When you or another user submit the exact same query or resource request again, the results will come back from the cache instead of from the data source itself.
```

Comment:
```suggestion
When you enable query and resource caching, Grafana temporarily stores the results of data source queries and resource requests. When you or another user submit the same query or resource request again, the results will come back from the cache instead of from the data source.
```